frontend/next.config.ts

```diff
@@ -1,7 +1,8 @@
 import type { NextConfig } from "next";
 
 const nextConfig: NextConfig = {
-  output: 'standalone',
-};
+  pageExtensions: ['js', 'jsx', 'ts', 'tsx', 'md'],
+  // Add any other Next.js config here
+}
```

Comment:
Looks like a rebase error, don't remove the old config

```suggestion
  pageExtensions: ['js', 'jsx', 'ts', 'tsx', 'md'],
  output: 'standalone',
}
```